kernel-laws/src/main/scala/cats/kernel/laws/IsSerializable.scala

```diff
@@ -19,7 +19,9 @@ private[laws] object IsSerializable {
       import java.io._
       val baos = new ByteArrayOutputStream()
       val oos = new ObjectOutputStream(baos)
+      // scalastyle:off null
       var ois: ObjectInputStream = null
+      // scalastyle:off on
```

Comment:
Thanks for finding another of my errors. I didn't know you could also specify the checker in the single line comment.